packages/bits/src/lib/wizard-v2/wizard.directive.ts

```diff
@@ -116,6 +135,22 @@ export class WizardDirective extends CdkStepper implements OnChanges, AfterConte
         super.ngOnDestroy();
     }
 
+    public getStepState(i: number, state: StepState = STEP_STATE.NUMBER): StepState {
+        const steps = this.steps.toArray();
+        const step = steps[i];
+        const isSelected = steps.indexOf(this.selected) === i;
+
+        if (step._showError && step.hasError && !isSelected) {
+            return STEP_STATE.ERROR;
+        } else if (isSelected) {
+            return STEP_STATE.EDIT
+        } else if (steps[i].completed) {
+            return STEP_STATE.DONE
+        } else {
+            return state;
+        }
```

Comment:
Maybe we don't need the `else` here:

```suggestion
        }
        
        return STEP_STATE.NUMBER;
```